ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -775,7 +774,7 @@ export const AdminInstitutionEdit = fp.flow(
           controlledTierEmailDomains: AccessTierShortNames.Controlled,
           requestAccessUrl: requestAccessUrl
             ? canonicalizeUrl(requestAccessUrl)
-            : '',
+            : null,
```

Comment:
interesting ... I would have thought that `requestAccessUrl && canonicalizeUrl(requestAccessUrl)` would work here.  But I tried it and it doesn't.  So this is a case where null and undefined work differently.

Maybe this is also related to line 882?